docs/getting_started/creating_an_admin.rst

```diff
@@ -83,7 +89,36 @@ two very simple ``Post`` and ``Tag`` entities will be used::
         {
             return $this->blogPosts;
         }
+        
+        /**
+         * Get id
+         *
+         * @return integer 
+         */
+        public function getId() {
+            return $this->id;
+        }
 
+        /**
+         * Get name
+         *
+         * @return string 
+         */
+        public function getName() {
+            return $this->name;
+        }
+
+        /**
+         * Set name
+         *
+         * @param string $name
+         * @return Category
+         */
+        public function setName($name) {
+            $this->name = $name;
+
+            return $this;
```

Comment:
Why?

Reply:
Hii Core23 I have changed protected to private.